pmml_recommendation/README.md

```diff
@@ -0,0 +1,5 @@
+# PMML Recommendation
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/kiegroup/trusty-ai-sandbox/tree/master/pmml-notebook/HEAD)
```

Comment:
Can you please check the URL? It seems not working

Reply:
@danielezonca The links should work after the merge. I'll look at them after it to verify/fix